docs/guides/admin/docs/configuration/multi.tenancy.md

```diff
@@ -41,15 +41,16 @@ rules. For this example we have a three node install of `admin.opencast.org`, `w
 `presentation.opencast.org`.  Assume that the new tenant is called `tenant1` and should be mapped to
 `tenant1-*.opencast.org`.
 
-### Tenant Configuration
+### Step 1: Tenant Configuration
 
 Create a file called org.opencastproject.organization-tenant1.cfg in the `etc/` directory of your Opencast
 installation, on each of the nodes.  As an example, this is what the admin node looks like:
 
     id=tenant1
     name=Tenant 1
-    server=tenant1-admin.opencast.org,tenant1-presentation.opencast.org
     port=8080
+    prop.org.opencastproject.host.admin.opencast.org=tenant1-admin.opencast.org
+    prop.org.opencastproject.host.presentation.opencast.org=tenant1-presentation.opencast.org
```

Comment:
@KatrinIhler would you make sure everything is correct here?

Reply:
Since we use `example.org` above, I guess we should also use it here

Reply:
The same thing comes up again in lines 58 and 59 after `# Admin and Presentation Server Urls`. I assume the change from `admin.opencast.org` (and `presentation.opencast.org`) to `[...].example.org` happens here as well?

Reply:
Yeah, let's be consistent. If you search for opencast.org, there are also a couple more instances in the text.

I don't really care either way, but we should pick one so users are less confused. :)